pkgs/top-level/python-packages.nix

```diff
@@ -4697,6 +4697,25 @@ in {
     };
   };
 
+  nginx-config-formatter = stdenv.mkDerivation rec {
+    version = "2016-06-16";
+    name = "nginx-config-formatter-${version}";
+
+    src = pkgs.fetchgit {
+      url = "git://github.com/1connect/nginx-config-formatter.git";
+      rev = "fe5c77d2a503644bebee2caaa8b222c201c0603d";
+      sha256 = "0akpkbq5136k1i1z1ls6yksis35hbr70k8vd10laqwvr1jj41bga";
+    };
+
+    installPhase = ''
+mkdir -p $out/bin
+cat <<EOF > $out/bin/nginxfmt
+#!${stdenv.shell}
+${pkgs.python3}/bin/python $src/nginxfmt.py "\$@"
+EOF
+chmod a+x $out/bin/nginxfmt
+'';
```

Comment:
makeWrapper, of course. D'oh.

But my original attempt was indented, and I ended up with EOF in the file. Not sure why; possibly tab/space confusion.

Don't have access to the machine this is on right now, though. I'll have another version up tomorrow.